includes/internal/emails/class-email-preview.php

```diff
@@ -127,8 +127,8 @@ public static function get_preview_link( int $post_id ): string {
 	 */
 	private function render_page(): void {
 		$subject      = $this->email_sender->get_email_subject( $this->get_email_post_for_preview(), $this->get_placeholders() );
-		$from_address = Sensei()->emails->get_from_address();
-		$from_name    = Sensei()->emails->get_from_name();
+		$from_address = $this->email_sender->get_from_address();
```

Comment:
This is a good place to encode special chars. Not inside getter.

Reply:
Those two filters that you mentioned above, as far as I know, don't use the values in the HTML context. [Here is documentation for the address hook.](https://developer.wordpress.org/reference/hooks/wp_mail_from/)
Not sure, if `esc_html` is better then the previous function, for HTML probably is.
But I want to highlight that's not sanitizing. 

Here is an example what it does: https://developer.wordpress.org/reference/functions/esc_html/#comment-595

It is what we want to do in HTML, yes, but I'd say that's incorrect to do for the email address when sending email.

What I want to say it that the "general purpose" function shouldn't do assumptions about how we use its data. We should apply proper handling in place: escape HTML before output (if needed), escape strings before inserting into DB, do proper kind of preparation of data in each case depending on context.

I know there is "a rule" about that all, but I failed to find it. However, I found another nice article that also mentions when we need to escape (`TL;DR Escape as late as possible`): https://docs.wpvip.com/technical-references/security/validating-sanitizing-and-escaping/

Reply:
Fixed here [a52a21a](https://github.com/Automattic/sensei/pull/6968/commits/a52a21a48d5962ce3a0084c0b11c4407386dbbd4) 
Reviewing, I found the preview.php is already running the  esc_html, so If we don't need it on preview and on the email. I just removed .